cmake/plugin.cmake

```diff
@@ -209,6 +209,11 @@ MACRO(MYSQL_ADD_PLUGIN)
 
     TARGET_LINK_LIBRARIES (${target} mysqlservices ${ARG_LINK_LIBRARIES})
 
+    IF(CMAKE_SYSTEM_NAME MATCHES AIX)
+      TARGET_LINK_OPTIONS(${target} PRIVATE "-Wl,-berok")
```

Comment:
You do not need a berok here, because you link to server shared library which exports everything you need, and has an import library, right?

Reply:
As it was on discussion, I have not send the commit yet. Now done, ``-berok`` is removed here.